class/defaults.yml

```diff
@@ -42,19 +42,19 @@ parameters:
               requests:
                 storage: 50Gi
       prometheusOperator: {}
+      prometheusOperatorAdmissionWebhook: {}
       alertmanagerMain:
         volumeClaimTemplate:
           spec:
             resources:
               requests:
                 storage: 2Gi
       kubeStateMetrics: {}
-      grafana: {}
```

Comment:
wait what?

Reply:
old old OCP used to have a grafana managed by the cluster-monitoring stack (<= 4.10 from clicking through the docs quickly)